<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphics</class>
 <widget class="QMainWindow" name="graphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>731</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlotAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>731</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSaveImage">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSaveOne">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save 1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSaveTwo">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save 2</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>390</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
